Add suggestion from Jonathan Hammell (#45)
cose-wg/X509: draft-ietf-cose-x509.md

  in the external_aad. As the end-entity certificate is integrity protected
  by COSE, the URI does not need to provide any protection.

  If a retrieved certificate does not chain to an existing trust anchor, that
  certificate MUST NOT be trusted unless the URI provided integrity protection
  and server authentication and the server is configured as trusted to provide
  new trust anchors or if an out-of-band confirmation can be received for trusting
  the retrieved certificate. In case an HTTP or CoAP GET request is used to
  retrieve a certificate, TLS {{RFC8446}}, DTLS {{I-D.ietf-tls-dtls13}} or

>   {{RFC8613}} SHOULD be used.

Reviewer: ```suggestion
  OSCORE {{RFC8613}} SHOULD be used.
```

>   {{RFC8613}} SHOULD be used.

Author: Done. Thanks!

provides guidance for the path validation. The security considerations
of {{RFC5280}} are also important for the correct usage of this document.

Protecting the integrity of the x5bag, x5chain and x5t contents by placing them

> in the protected header bucket MAY mitigate some risks of a misbehaving

Reviewer 2: This is not a proper use of RFC 2119 MAY.

```suggestion
in the protected header bucket can help mitigate some risks of a misbehaving
```

> certificate authority (c.f. {{Section 5.1 of RFC2634}}).

Reviewer 2: c.f. ➔ cf.

Author: Done. Thanks!

> certificate authority (c.f. {{Section 5.1 of RFC2634}}).

Reviewer 2: ```suggestion
certificate authority (cf. {{Section 5.1 of RFC2634}}).
```
